packages/odyssey-design-tokens/src/layer.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
This file was unused, so I've removed it and brought in a `layout` file for `max-line-length`.